docs/pages/mainpage.md

```diff
@@ -2,13 +2,15 @@
 
 ## Introduction
 
-sACN is a C-language library that implements **ANSI E1.31: Lightweight streaming protocol for
-transport of DMX512 using ACN**, commonly referred to as **Streaming ACN** or **sACN**. The sACN
-library is designed to be portable and scalable to almost any sACN usage scenario, from lightweight
-embedded devices to large-scale data sending operations.
+*Streaming ACN (sACN)* is an ANSI standard for entertainment technology by
```

Comment:
This seems like it was fine before? It leads with a description of what's in this repo, not what the standard is.

Reply:
Ok.  I can revert that, but add "(with C++ wrappers)".

Reply:
Sure... or take the [EtcPal approach](https://etclabs.github.io/EtcPal/docs/v0.3/) and mention the C++ wrappers later. Or just say "a library"